syncmaster/backend/api/v1/runs.py

```diff
@@ -104,6 +107,10 @@ async def start_run(
         transfer.target_connection_id,
     )
 
+    request_correlation_id = correlation_id.get()
+    if not request_correlation_id:
+        request_correlation_id = str(uuid.uuid4())
```

Comment:
How does this value is being passed to Celery worker?

Reply:
added logging settings for worker where correlation is formatted into log message